file/file-alfresco/src/main/java/uk/gov/justice/services/file/alfresco/sender/AlfrescoFileSender.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.justice.services.file.alfresco.sender;
+
+import static javax.ws.rs.client.Entity.entity;
+import static javax.ws.rs.core.MediaType.MULTIPART_FORM_DATA_TYPE;
+
+import uk.gov.justice.services.common.configuration.GlobalValue;
+import uk.gov.justice.services.file.alfresco.rest.AlfrescoRestClient;
+import uk.gov.justice.services.file.api.sender.FileData;
+import uk.gov.justice.services.file.api.sender.FileSender;
+
+import java.util.Collections;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.ws.rs.core.MultivaluedHashMap;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@ApplicationScoped
+public class AlfrescoFileSender implements FileSender {
+
+    private static final String CPPUID = "cppuid";
```

Comment:
This is what the material context is sending so I presume it's been configured somewhere in Alfresco